api/observability/v1/output_types.go

```diff
@@ -769,3 +774,40 @@ type Syslog struct {
 	// +kubebuilder:validation:Optional
 	MsgID string `json:"msgID,omitempty"`
 }
+
+type OTLPTuningSpec struct {
+	BaseOutputTuningSpec `json:",inline"`
+
+	// Compression causes data to be compressed before sending over the network.
+	// It is an error if the compression type is not supported by the output.
+	//
+	// +kubebuilder:validation:Enum:=gzip;none
+	// +kubebuilder:default:=gzip
+	Compression string `json:"compression,omitempty"`
+}
+
+// OTLP provides configuration for sending logs via OTLP/HTTP requests with JSON-encoded payloads.
```

Comment:
OTLP defines configuration for sending logs via OTLP using OTEL scemantic conventions
https://opentelemetry.io/docs/specs/otlp/#otlphttp